src/lib/viewers/BaseViewer.js

```diff
@@ -1100,6 +1101,18 @@ class BaseViewer extends EventEmitter {
         this.emit('annotatorevent', data);
     }
 
+    handleAnnotationCreateEvent({ annotation, meta: { status } = {} }) {
+        if (status !== 'pending') {
```

Comment:
What would the experience be on a failure? Would the unsaved annotation be lost?

Reply:
That's a good question, according to the designs, it looks like the failed annotation remains -- I'll follow up with design